webcommon/cordova.platforms.ios/licenseinfo.xml

```diff
@@ -21,6 +21,13 @@
 -->
 <licenseinfo>
     <fileset>
+        <!-- Test data, should not contain comments -->
+        <file>cordova.platforms.ios/test/unit/src/org/netbeans/modules/cordova/platforms/ios/disconnected.xml</file>
+        <file>cordova.platforms.ios/test/unit/src/org/netbeans/modules/cordova/platforms/ios/jsoncommand.xml</file>
+        <file>cordova.platforms.ios/test/unit/src/org/netbeans/modules/cordova/platforms/ios/listing.xml</file>
+        <file>cordova.platforms.ios/test/unit/src/org/netbeans/modules/cordova/platforms/ios/toback.xml</file>
+        <file>cordova.platforms.ios/test/unit/src/org/netbeans/modules/cordova/platforms/ios/tofront.xml</file>
```

Comment:
Possibly `type="CATEGORY_B"` could be more appropriate ("The contained code is used at runtime and must be included in source form"). 
IMHO it's better to handle/declare "external" licenses for specific files locally rather than in global `rat-exclusions.txt`
.

Reply:
I would just add the ALv2 header and be done with it. No need to exclude the files. The exception you could pull here (https://www.apache.org/legal/src-headers.html#faq-exceptions) is:

> Other files may make sense to have no license header. [...]
> - Test data for which the addition of a source header would cause the tests to fail. 
> [...]

That is not the case here. The unittests are not position dependend and as far as I can see they use an XML parser to process the XML data, so a normal XML comment should not hurt.

Reply:
OK. Someone with IOS access should verify the tests still work :)